credit-conveyor/Deal/src/main/java/com/deals/feignClient/ClientSide.java

```diff
@@ -0,0 +1,26 @@
+package com.deals.feignClient;
+
+import com.deals.dto.CreditDTO;
+import com.deals.dto.LoanApplicationRequestDTO;
+import com.deals.dto.LoanOfferDTO;
+import com.deals.dto.ScoringDataDTO;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.util.List;
+
+@FeignClient(url = "https://localhost:8080")
```

Comment:
Good way is to put such properties as urls in application.properties file, and inject them from there.